### ISSUE TITLE
Investigate pull-etcd-verify golangci-lint not found

### ISSUE THREAD

**Issue by ArkaSaha30** [etcd-io/etcd#17637]
### Which Github Action / Prow Jobs are flaking?

Prowjob:
- pull-etcd-verify

https://github.com/kubernetes/test-infra/blob/master/config/jobs/etcd/etcd-presubmits.yaml#L53-L87

### Which tests are flaking?

Prowjob:
- pull-etcd-verify

### Github Action / Prow Job link

https://testgrid.k8s.io/sig-etcd-presubmits#pull-etcd-verify

### Reason for failure (if possible)

- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-etcd-verify/1769284830702342144
```
stderr: ./scripts/test_lib.sh: line 148: golangci-lint: command not found
```
- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-etcd-verify/1768163273623998464
- https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/directory/pull-etcd-verify/1768162736987967488
```
CONFLICT (content): Merge conflict in tests/integration/v3_watch_test.go
Automatic merge failed; fix conflicts and then commit the result.
```

**Comment by ArkaSaha30** [comment]
/assign

cc @jmhbnz @siyuanfoundation

**Comment by jmhbnz** [comment]
/retitle Investigate pull-etcd-verify golangci-lint not found

**Comment by ivanvc** [comment]
This was fixed by kubernetes/test-infra#32563, sample success run: https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/etcd-io_etcd/17935/pull-etcd-verify/1787230289815146496. 
Can we close this issue now?
